backend/pkg/api/instances.go

```diff
@@ -631,3 +639,70 @@ func (api *API) instanceStatusHistoryQuery(instanceID, appID, groupID string, li
 		Order(goqu.C("created_ts").Desc()).
 		Limit(uint(limit))
 }
+
+// instanceFactQuery returns a SelectDataset prepared to return all instances
+// that have been checked in the last hour.
+func (api *API) instanceFactQuery() *goqu.SelectDataset {
+	return goqu.From("instance_application").
+		Select(
+			goqu.C("last_check_for_updates").As("timestamp"),
+			goqu.C("c.name").As("channel_name"),
+			goqu.Case().
+				When(goqu.C("c.arch").Eq(1), "AMD64").
+				When(goqu.C("c.arch").Eq(2), "ARM").
+				As("arch"),
+			goqu.C("version"),
+			goqu.L("COUNT(*)").As("instances")).
+		Join(goqu.T("groups"), goqu.On(goqu.C("group_id").Eq(goqu.C("groups.id")))).
+		Join(goqu.T("channel"), goqu.On(goqu.C("groups.channel_id").Eq(goqu.C("channel.id")))).
+		Join(goqu.T("package"), goqu.On(goqu.C("channel.package_id").Eq(goqu.C("package.id")))).
+		Where(
+			goqu.C("last_check_for_updates").Gte(goqu.L("now() - interval '1 hour'")),
+			goqu.C("last_check_for_updates").Lte(goqu.L("now()"))).
+		GroupBy("last_check_for_updates", "c.name", "c.arch", "version")
+}
+
+func (api *API) GetInstanceFact() ([]InstanceFact, error) {
```

Comment:
```suggestion
func (api *API) GetInstanceFacts() ([]InstanceFact, error) {
```
Let's use the plural as it returns many